tests/unit_tests/test_mcda_with_robustness.py

```diff
@@ -1,28 +1,32 @@
 import pandas as pd
 import numpy as np
 import unittest
-from unittest import TestCase
 
-from mcda.mcda_with_variability import MCDAWithVar
+from mcda.mcda_with_robustness import MCDAWithRobustness
 from mcda.configuration.config import Config
 
 
-class TestMCDA_with_variability(unittest.TestCase):
+class TestMCDA_with_robustness(unittest.TestCase):
```

Comment:
Maybe I missed before, is there any convention to use just `camelCase` for class names and `_` in the name of test classes?
Can we have same formatting for class names irrespective of test or a normal class? Someone might confuse test class names with `_` to a variable name.

Reply:
Apparently, in Python one can use underscores to separate words in class names and in test class names, following the convention known as "snake_case", as well as the CamelCase convention. Functions instead should be defined by using the "snake_case" convention only. Classes and functions cannot easily be confused because classes are always preceded by the word `class`.